csharp/src/Apache.Arrow/Arrays/BooleanArray.cs

```diff
@@ -34,11 +161,8 @@ public BooleanArray(ArrayData data)
 
         public override void Accept(IArrowArrayVisitor visitor) => Accept(this, visitor);
 
-        public bool? GetBoolean(int index)
```

Comment:
Non nullable anymore? DateTime below still seems to return a nullable? 

In any case, the last time I checked, null support was not complete, so this is not really an important concern yet.

Reply:
I've opened https://issues.apache.org/jira/browse/ARROW-5708 for this.